### PR TITLE
NL translations and punctuation improvements in the translation strings.

### PR DESCRIPTION
**Pull request by ikmaak**
Apply only after 0.7.1 is released. Sorry for not knowing about the multiple commits, will learn more about feature branches soon :)

### CODE REVIEW

File: plinth/modules/tor/__init__.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def _diagnose_control_port():
 def _diagnose_url_via_tor(url, kind=None):
     """Diagnose whether a URL is reachable via Tor."""
     result = action_utils.diagnose_url(url, kind=kind, wrapper='torsocks')
-    result[0] = _('Access URL {url} on tcp{kind} via Tor') \
+    result[0] = _('Access URL {url} on tcp {kind} via Tor') \
```
Comment (SunilMohanAdapa) [inline plinth/modules/tor/__init__.py]: The outcome of the string should be 'tcp4' or 'tcp6'.  So, the original is correct.

**Comment by SunilMohanAdapa** [comment]
There are three strings that are titles of pages: 'Edit Connection', 'Delete User', 'Edit User'.  Removing their title casing may not entirely be consistent with other titles. I understand that you probably did that because of the name following these titles. 

- For 'Edit Connection/User X' we can simply have the title 'X' like desktop user interfaces do with many dialogs.
- For 'Delete Connection/User X' we could keep them title cased and remove the connection/user name from the title and add the it to the statement below that warns of a delete (perhaps in bold)?

**Comment by SunilMohanAdapa** [comment]
@ikmaak Do my suggestions look okay?  I can make the necessary changes before merge.

**Comment by ikmaak** [comment]
suggestions seem ok to me :)

**Comment by SunilMohanAdapa** [comment]
No problem with the combined commits in a pull request.  I have merged the commits with the following changes:

- "Edit/Delete Connection/User XYZ" has been changed as discussed.
- "tcp4/6" change ignored as discussed.
- "Firewall Zone" instead of "Firewall zone" in the left column of the table will make the page inconsistent.  That will be only such item to be title cased in the page.  Since each of them are not section headers, lower case is more correct.
- "Edit Connection" and "Delete Connection" should not be title cased because they are part of tooltips.
- "Change Password" should not be title cased because it is part of menu and changing the single item will make it inconsistent with "Log in", "Log out" etc.  Changing this is okay if all the other menu items are also changed similarly.

I am closing this pull request.  Please feel free to reopen for further discussion and changes.

**Pull request closed without merging**